tests/ut_seattlelibv2_repypp_include.py

```diff
@@ -0,0 +1,43 @@
+"""
+repypp.py is a preprocessor for repy. It includes dependent files as needed.This is used to help the 
+programmer avoid the need to use import. They can instead use "include X" which works somewhat like "from X import *".
```

Comment:
Please limit line lengths to approximately 70 characters. (This eases editing on text-based terminals.)